README.md

```diff
@@ -9,12 +9,35 @@
 [codecov]: https://codecov.io/github/cosmos/solidity-ibc-eureka
 [codecov-badge]: https://codecov.io/github/cosmos/solidity-ibc-eureka/graph/badge.svg?token=lhplGORQxX
 
-This is a work-in-progress IBC Eureka implementation in Solidity. IBC Eureka is a simplified version of the IBC protocol that is encoding agnostic. This project also includes an [SP1](https://github.com/succinctlabs/sp1) based tendermint light client for the Ethereum chain, and a POC relayer implementation.
+This is a work-in-progress IBC Eureka implementation in Solidity. IBC Eureka is a simplified version of the IBC protocol that is encoding agnostic. This enables a trust-minimized IBC connection between ethereum and a Cosmos SDK chain.
 
 ## Overview
 
 `solidity-ibc-eureka` is an implementation of IBC in Solidity.
 
+- [IBC Eureka in Solidity     ](#ibc-eureka-in-solidity-----)
```

Comment:
I tried this, and it really helps with the organization of the README. Great work!